tutorials/integration.md

```diff
@@ -268,16 +311,21 @@ all the gas of the transaction, except for the unknown amount we are going to us
         emit PostRelayed(abi.decode(context, (uint)));
     }
 ```
-This function returns the version of the paymaster contract. 
+
+&nbsp;
+
+This function returns the version of the GSN protocol the paymaster supports.
+In this case, the version is the latest at writing, `2.0.3`.
+
 ```solidity
     function versionPaymaster() external virtual view 
     override returns (string memory) {
-        return "1.0";
+        return "2.0.3";
```

Comment:
great to fix this. this is a bug we had in our documentation, which broke the implementation of users...